.github/workflows/third-party-library-changes.yml

```diff
@@ -0,0 +1,78 @@
+name: Third Party Library Changes
+on:
```

Comment:
I think it would be good to also run this on `push` events to `trunk` and numbered branches, just in case something sneaks in that is not tested through a pull request.